lib/index.js

```diff
@@ -39,11 +38,8 @@ exports.register = (server, options, next) => {
         method: internals.onPreStart(monitor, result.value)
     }]);
 
-    return monitor.start(next);
+    monitor.start();
 };
 
 
-exports.register.attributes = {
-
-    pkg: require('../package.json')
-};
+exports.pkg = require('../package.json');
```

Comment:
Why this change?

Reply:
The plugin format has changed without it you get this error
```
[1] "name" is required
      at Object.exports.apply (/root/node_modules/hapi/lib/config.js:22:10)
      at internals.Server.register (/root/node_modules/hapi/lib/server.js:384:31)
      at it (/root/test/index.js:70:22)
      at Immediate.setImmediate (/root/node_modules/lab/lib/runner.js:566:31)
      at runCallback (timers.js:800:20)
      at tryOnImmediate (timers.js:762:5)
      at processImmediate [as _immediateCallback] (timers.js:733:5)
```

Reply:
Hmmm, you're sure that this works? According to the error message you have to provide `exports.name` in addition.

Reply:
@frankthelen Yes I am sure the error was provided simply by switching it back and and running the test suit to see it fail. pkg can be used as well please see the very last paragraph and code example here: https://hapijs.com/api

Reply:
Cool. You're right.